docs/SETUP_GUIDE.md

```diff
@@ -4,48 +4,108 @@ Here are the steps required to setup an instance of the rolling demo on your own
+- [kubectl](https://kubernetes.io/docs/reference/kubectl/)
 
-- You can create the token either [via the CLI](https://argo-cd.readthedocs.io/en/latest/user-guide/commands/argocd_account_generate-token/), or, by going to argocd on your cluster > `settings` > `accounts` > `admin` > `Generate New` (token) and generate a token for admin. This token should be given to the `ARGOCD_API_TOKEN` variable.
+#### Setup the `private-env` file
 
-- Note, if you need to register and authenticate this github repo in your argoCD instance by going to > `settings` > `repositories` > `Connect Repo`.
+- Create a file called `private-env` inside the [scripts/](./scripts/) directory and copy all the contents of the [scripts/env](./scripts/env) file there.
 
-#### Create the `private-env` file
+- Your `scripts/private-env` should look like:
 
-- Create a file called `private-env` inside the [scripts/](./scripts/) directory and copy all the contents of the [scripts/env](./scripts/env) file there.
-- Populate all the variables mentioned in the file with your secrets.
+```bash
+# Github secrets
+# For more information on how to setup your github app
+# you can take a look here:
+# https://github.com/redhat-ai-dev/ai-rhdh-installer/blob/main/docs/APP-SETUP.md
+export GITHUB_APP_APP_ID="your-github-apps-app-id"
+export GITOPS_GIT_TOKEN="your-github-token"
+export GITOPS_GIT_ORG="your-github-org-name"
+export GITHUB_APP_CLIENT_ID="your-github-app-client-id"
+export GITHUB_APP_CLIENT_SECRET="your-github-app-client-secret"
+export GITHUB_APP_WEBHOOK_URL="your-github-app-webhook-url"
+export GITHUB_APP_WEBHOOK_SECRET="your-github-app-webhook-secret"
+export GITHUB_APP_PRIVATE_KEY="your-github-org-name"
+
+# ArgoCD secrets
+export ARGOCD_USER="your-user" # default value is "admin"
+
+# RHDH secrets
+export BACKEND_SECRET="a-randomly-generated-string"
+# RHDH_CLUSTER_ROUTER_BASE: should contain the domain suffix of all the
+# OCP routes.
+export RHDH_CLUSTER_ROUTER_BASE="apps.mycluster.openshift.com"
+export RHDH_BASE_URL="https://rolling-demo-backstage-rolling-demo-ns.${RHDH_CLUSTER_ROUTER_BASE}" # this is the route for the RHDH console
+export RHDH_CALLBACK_URL="${RHDH_BASE_URL}/api/auth/oidc/handler/frame"
+
+# RHDH Postgres Secrets - any valid password strings for these two env vars will do
+export POSTGRESQL_POSTGRES_PASSWORD="your-preffered-postgres-pass"
+export POSTGRESQL_USER_PASSWORD="your-preffered-user-pass"
+
+# Quay.io secrets
+# For more information on how to setup quay.io you check the doc
+# here: https://github.com/redhat-ai-dev/ai-rhdh-installer/blob/main/docs/APP-SETUP.md#quay-setup
+export QUAY_DOCKERCONFIGJSON="your-quay.io-dockerconfig.json"
+
+# KeyCloak (RH SSO) secrets
+# In this area you can point to an already existing keycloak instance.
+# As per RHDHPAI use case please refer to the already existing secrets in our Dev Instance.
+export KEYCLOAK_METADATA_URL="{your-instance's-url}/auth/realms/your-realm"
```

Comment:
let's supply the actual URL on our dev cluster, and then put a comment above  to explain what you change if you point to another keycloak instance